tests/acceptance/stepDefinitions/sharingContext.js

```diff
@@ -742,6 +777,17 @@ When(
   }
 )
 
+When('the user edits share with user/group {string} of file/folder/resource {string} changing following',
```

Comment:
ìn phoenix its called "collaborator" not "share" and  please add "using the webUI" to the sentence

Reply:
done!